config/database.yml

```diff
@@ -1,6 +1,6 @@
 default: &default
   adapter:  postgresql
-  host:     localhost
```

Comment:
Ik, and it's ok if you run the app using docker-compose since the container name becomes host name.

But either way, IMO it should be taken from some env var

Reply:
Not really. The database is run on localhost and this where it accepts requests. In development the database is run on localhost.

Reply:
Setting `DATABASE_URL` ENV overrides those settings, so I'd suggest leaving localhost here and setting `DATABASE_URL` in docker-compose. That way, it's still easy to develop locally outside the docker, we don't couple config files with docker hosts names, and it'll still work with a dockerized setup